@xen-orchestra/lite/src/components/AppTooltip.vue

```diff
@@ -18,29 +18,19 @@ const props = defineProps<{
 
 const tooltipElement = ref<HTMLElement>();
 
-const content = computed(() =>
-  isString(props.options) ? props.options : props.options.content
-);
-
 const isDisabled = computed(() => {
-  if (isEmpty(content.value)) {
+  if (props.options.content === false) {
     return true;
   }
 
-  if (isString(props.options)) {
-    return false;
-  }
-
-  if (isFunction(props.options.disabled)) {
-    return props.options.disabled(props.target);
+  if (isString(props.options.content)) {
+    return props.options.content.trim() === "";
   }
 
-  return props.options.disabled ?? false;
+  return false;
```

Comment:
It's not really important but, how about a shorter version instead?

```ts
const isDisabled = computed(() => 
   isString(props.options.content) ? props.options.content.trim() === "" : props.options.content === false
)
```

Reply:
I've been a developer for more than 20 years, but I still wonder when to use concise code or explicit code 😅 

I'll go for your suggestion here thought 👍🏻